# 磁铁位姿信息
# 时间戳
Header header

# 磁铁中心的位置
geometry_msgs/Point position

# 磁铁的方向（四元数表示）
geometry_msgs/Quaternion orientation

# 磁铁的强度（mT）
float64 magnetic_strength
